backend/de.metas.adempiere.adempiere/base/src/main/java-gen/org/compiere/model/I_C_Invoice.java

```diff
@@ -1971,4 +1971,33 @@
     public static final org.adempiere.model.ModelColumn<I_C_Invoice, Object> COLUMN_UserFlag = new org.adempiere.model.ModelColumn<I_C_Invoice, Object>(I_C_Invoice.class, "UserFlag", null);
     /** Column name UserFlag */
     public static final String COLUMNNAME_UserFlag = "UserFlag";
+    
+    
+    /**
+	 * Set Lager.
+	 * Storage Warehouse and Service Point
+	 *
+	 * <br>Type: TableDir
+	 * <br>Mandatory: true
```

Comment:
why is the Warehouse mandatory?

Reply:
Copy - paste problem.
Regenerated the classes and hey are now ok.